zhaquirks/sinope/switch.py

```diff
@@ -123,7 +142,6 @@ class SinopeTechnologiesLoadController(CustomDevice):
         # output_clusters=[3, 4, 25]>
         MODELS_INFO: [
             (SINOPE, "RM3250ZB"),
-            (SINOPE, "RM3500ZB"),
```

Comment:
Are you sure that there is no `RM3500ZB` devices with that signature.
Because you have added a new device signature there's no need to remove it from others quirks just because have the same model. While the signatures are different you can keep both of them.